From 5ea245d057d66c81ff4cc44e5510260e4d79a98f Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Tue, 22 May 2018 15:33:54 +0800
Subject: [PATCH 3871/5242] MLK-18368-7: imx-hdmi: change the entry id of meta
 data

commit  cf85ee55abb287b11a58c2da5db413c7f415ddd9 from
https://source.codeaurora.org/external/imx/linux-imx.git

For entry id of video is 0,  audio is 1, if entry id of meta data is
1, then it conflict with audio.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-hdmi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/imx-hdmi.c b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
index 786b235..52c1079 100644
--- a/drivers/gpu/drm/imx/hdp/imx-hdmi.c
+++ b/drivers/gpu/drm/imx/hdp/imx-hdmi.c
@@ -399,7 +399,7 @@ int hdmi_write_hdr_metadata(state_struct *state,
 	buffer[0] = 0;
 	infoframe_size++;
 
-	return CDN_API_InfoframeSet(state, 1, infoframe_size,
+	return CDN_API_InfoframeSet(state, 2, infoframe_size,
 				    (u32 *)buffer,
 				    HDMI_INFOFRAME_TYPE_DRM);
 }
-- 
1.7.9.5

